opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/UserManager.java

```diff
@@ -771,7 +771,7 @@ public AuthenticationResponse login(String organizationId, String username, Stri
         auditManager.auditUser(organizationId, username, Enums.Action.LOGIN, username,
                 new AuditRecord.Status(AuditRecord.Status.Result.SUCCESS));
         String userId = authenticationFactory.getUserId(organizationId, authId, response.getToken());
-        if (!CatalogAuthenticationManager.INTERNAL.equals(authId)) {
+        if (!CatalogAuthenticationManager.OPENCGA.equals(authId) && !CatalogAuthenticationManager.INTERNAL.equals(authId)) {
```

Comment:
Sometimes you're checking against both "opencga" and the old "internal", and some others just against "opencga"... is it justified every time?
Why not adding a method and check both always?

Reply:
INTERNAL should only be checked here. I'm not checking INTERNAL anywhere else. If the installation hasn't been migrated yet and an "opencga managed" user attempts to log in, the authId for that user will still be "internal". Once it has been migrated, the auth origin for all "opencga managed" users will then become "OPENCGA".